examples/self_weight_clamped_truss/self_weight_clamped_truss.jl

```diff
@@ -23,20 +23,20 @@ weight deformed configuration.
 #-----------------------------
 =#
 using ONSAS, Test
-using Dictionaries: dictionary
+using Dictionaries: Dictionary, dictionary
 
 "Return problem parameters"
 function parameters()
-    N = 10     # Number of elements.
+    N = 4     # Number of elements.
     E = 30e6    # Young's modulus.
     ν = 0.3     # Poisson's ratio.
-    ρ = 750.0   # Density.
+    ρ = 400.0   # Density.
     L = 200     # Element length.
     A = 1       # Cross section area.
     F = 10e6    # Force at the tip
     g = 9.81    # Gravity
     ϵ_model = RotatedEngineeringStrain
-    (; g, N, E, ν, ρ, L, A, F, ϵ_model)
+    g, N, E, ν, ρ, L, A, F, ϵ_model
```

Comment:
I see that you remove the named tuple here, is it on purpose?

Reply:
Was a typo, now is fixed